app/test/package/backend_test.dart

```diff
@@ -480,6 +482,56 @@ void main() {
           // TODO: check history
           // TODO: check assets
         });
+
+        testWithServices('package under publisher', () async {
```

Comment:
Couldn't we cover this by hitting the API? thus, creating a test that we don't need to update when we migrate away from `pub_server`?

Reply:
We could, but it is much more work to get all the required boilerplate setup and going. `pkg/pub_integration` tests do that, not sure if we really need to have that kind of setup for all the tests.